clients/ui/bff/internal/mocks/static_data_mock.go

```diff
@@ -273,6 +273,18 @@ func catalogCustomProperties() *map[string]openapi.MetadataValue {
 				MetadataType: "MetadataStringValue",
 			},
 		},
+		"tensorType": {
```

Comment:
This should be snake case `tensor_type` like all the other properties in this file that has more than one word.